BZ2110179: Update UI procedures for top-level Virt (#48203)
openshift/openshift-docs: modules/virt-initiating-vm-migration-web.adoc

[NOTE]
====

> The *Migrate* action is visible to all users but only admin users can initiate a virtual machine migration.

Reviewer: ```suggestion
The *Migrate* action is visible to all users, but only admin users can initiate a virtual machine migration.
```

====

.Procedure

. In the {VirtProductName} console, click *Virtualization* -> *VirtualMachines* from the side menu.

. You can initiate the migration from this page, which makes it easier to perform actions on multiple virtual machines on the same page, or from the *VirtualMachine details* page where you can view comprehensive details of the selected virtual machine:

> * Click the Options menu {kebab} at the end of virtual machine and select *Migrate*.

Reviewer: ```suggestion
* Click the Options menu {kebab} next to the virtual machine and select *Migrate*.
```
